app/models/invoice.rb

```diff
@@ -322,6 +322,14 @@ def should_sync_sales_order?
       end
   end
 
+  def document_invoice_name
+    return I18n.t('invoice.prepaid_credit_invoice') if credit?
+
+    return I18n.t('invoice.document_tax_name') if %w[AU AE ID NZ].include?(organization.country)
```

Comment:
what about cases where the customer is in that country?

Reply:
This spec is covering just organization country and we will see if it satisfy customer needs. In general, there are bunch of other conditions, like customer country or thresholds which can define if to present tax invoice or not. However, we will tackle it in another iteration if there will be need for it...

Reply:
okay 👍